src/item.cpp

```diff
@@ -1229,6 +1230,11 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
             info.emplace_back( "FOOD", _( "Vitamins (RDA): " ), required_vits );
         }
 
+        if( u.allergy_type( *food_item ) != morale_type( "morale_null" ) ) {
+            info.emplace_back( "DESCRIPTION",
+                               _( "* This food will cause you an <bad>allergic reaction</bad>." ) );
```

Comment:
```suggestion
                               _( "* This food will cause an <bad>allergic reaction</bad>." ) );
```

Reply:
In that case, change it to `This food will cause you to have an allergic reaction`. What you have is technically correct English, but I would never say it that way.

Reply:
The last one sounds nice, but I was trying really hard to keep it as short as possible to avoid having the sentence split in two lines due to length.